db/migrate/20230309001620_create_posts.rb

```diff
@@ -1,6 +1,6 @@
 class CreatePosts < ActiveRecord::Migration[7.0]
   def change
-    create_table :posts do |t|
+    create_table :posts true do |t|
```

Comment:
<img width="1302" alt="image" src="https://user-images.githubusercontent.com/65068771/224864470-2ae6159f-fdeb-4157-8416-82e856e23382.png">

- [ ] Kindly remove the keyword `true` here as it is preventing your migration to run.

Reply:
resolved! ✔️